transports/jms/src/main/java/org/mule/transport/jms/activemq/ActiveMQJmsConnector.java

```diff
@@ -53,6 +53,15 @@ protected void applyVendorSpecificConnectionFactoryProperties(ConnectionFactory
     {
         try
         {
+            Method getPrefetchPolicy = connectionFactory.getClass().getMethod("getPrefetchPolicy");
```

Comment:
I need to use reflection because ConnectionFactory is the javax class. it's the same reason why I must use reflection to get getRedeliveryPolicy.